Lib/test/test_typing.py

```diff
@@ -2802,6 +2802,14 @@ def __init__(self, attr: T) -> None:
         self.assertEqual(deepcopy(ci).attr, 1)
         self.assertEqual(ci.__orig_class__, C[int])
 
+    def test_copy_with(self):
+        # bpo-46581
+        from typing import Callable, ParamSpec
```

Comment:
Aren't they imported at the top of the file?

Reply:
Yeah, fair. Removed. (These were residual from originally placing this test in `test_genericalias.py`.)